server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +622,34 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to room from broadcast-signal endpoint
+			if (httpServer !== undefined && eventEmitter !== undefined) {
```

Comment:
Do we need `httpServer !== undefined` (or as a parameter at all)? Not used in the new code.

Reply:
Also, if both parameters are always used together, (Like there is no point to having one be defined and the other undefined then they should both live within a single optional paremeter like 
```
XXXConfig?: {
	httpServer: core.IHttpServer,
	eventEmitter: EventEmitter,
}
```

Reply:
> Do we need httpServer !== undefined (or as a parameter at all)? Not used in the new code.

Good catch. Yes this hasn't been needed for a while! Removing!